src/roll.ts

```diff
@@ -62,12 +62,4 @@ export default function apply(ctx: Context, options: RollConfig = {}) {
       }
       return output
     })
-
-  ctx.middleware((session, next) => {
```

Comment:
why remove middleware? this will disable common shortcut like `.rd6`

Reply:
Oh, goddamn it. Forgive me for forgetting such an important usage. I'll add it later.